### PR TITLE
Add `androidHardwareAccelerationDisabled` prop

### PR DESCRIPTION
**Pull request by YangXiaomei**
Hardware acceleration is enabled by default if your Target API level is >=14. However, because hardware acceleration is not supported for all of the 2D drawing operations, turning it on might affect some of your custom views or drawing calls. To disable hardware acceleration in application level will slow down the app, so we'd better to add a prop to enable user to disable hardware acceleration for android platform in view level
 Please view the details in [Hardware acceleration](https://developer.android.com/guide/topics/graphics/hardware-accel#controlling)

### CODE REVIEW

File: android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java [hunk 1]
```diff
@@ -529,6 +529,13 @@ public void setJavaScriptEnabled(WebView view, boolean enabled) {
     view.getSettings().setJavaScriptEnabled(enabled);
   }
 
+  @ReactProp(name = "androidHardwareAccelerationDisabled")
+  public void setHardwareAccelerationDisabled(WebView view, boolean disabled) {
+    if (disabled) {
+      view.setLayerType(View.LAYER_TYPE_SOFTWARE, null);
+    }
```
Comment (Titozzz) [inline android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java]: maybe re-enable it in an else case ?

Comment (YangXiaomei) [reply android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java]: Thanks for your suggestion, from the guide [
Hardware acceleration](https://developer.android.com/guide/topics/graphics/hardware-accel#controlling)  currently user cannot enable hardware acceleration at the view level.

Comment (Titozzz) [reply android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java]: you are 100% right

**Comment by Titozzz** [comment]
@allcontributors[bot] please add @YangXiaomei for code, doc

**Pull request merged**